### DISCUSSION TITLE
Issue with permissions

### DISCUSSION THREAD

**Discussion by GibonIT** [amidaware/tacticalrmm#1409] [General]
Hello all,
I think there is a little issue with permissions. I have created a role with permissions to only view a single client and his site, but when I logon with that user, I can see more clients in the tree. I can do nothing in the clients that I dont have permissions to, but I think from a security/privacy perspective, I should not even see the other clients listed at all.

Thanks for feedback
Leos

**Comment by wh1te909** [comment]
Please share a screenshot of the user's permissions.

**Comment by GibonIT** [comment]
![image](https://user-images.githubusercontent.com/93908808/213858287-8532adb0-22ea-4508-a6a6-29e57be2d787.png)
and I also see different customers in the list.

**Reply by wh1te909** [reply]
https://user-images.githubusercontent.com/7434746/213860894-31a237e8-f746-4567-bcc5-3cd6f0821e36.mp4

Please watch this video. I am not seeing any issues with permissions and it's working as intended. Maybe you need to refresh your browser cache. 

Also if you want to give access to 1 or more sites, leave the "Allowed Clients" field blank and only give access to the sites. If you give access to a client, that will overwrite site permissions and give access to all sites in that client.

**Reply by GibonIT** [reply]
Your right, it was my fault, I didnt have super user assigned to anyone. Now all works well. Thanks
